.github/workflows/main.yaml

```diff
@@ -364,6 +380,87 @@ jobs:
           kubectl wait --for=delete deployment/test-chart --timeout=2m
           kubectl wait --for=delete deployment/replicated --timeout=2m
 
+      # validate status informers
+      - name: Create empty status informers for validation
+        run: |
+          cat << EOF > test-values.yaml
+          replicated:
+            statusInformers: []
+          EOF
+    
+      - name: Install via Helm as subchart in production mode and pass empty status informers
+        run: |
+          helm install test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false -f test-values.yaml --wait --timeout 2m
+
+          COUNTER=1
+          while ! kubectl logs deploy/replicated | grep -qv 'Generated informers from Helm release'; do
+            ((COUNTER += 1))
+            if [ $COUNTER -gt 60 ]; then
+              echo "Did not receive empty status informers"
+              kubectl logs deploy/replicated
+              exit 1
+            fi
+            sleep 1
+          done
+
+      - name: Upgrade via Helm as subchart in production mode to use default status informers
+        run: |
+          helm upgrade test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false --wait --timeout 2m
+
+          COUNTER=1
+          while ! kubectl logs deploy/replicated | grep -q 'Generated informers from Helm release'; do
+            ((COUNTER += 1))
+            if [ $COUNTER -gt 60 ]; then
+              echo "Did not receive default status informers"
+              kubectl logs deploy/replicated
+              exit 1
+            fi
+            sleep 1
+          done
+
+      - name: Uninstall test-chart via Helm
+        run: helm uninstall test-chart --wait --timeout 2m
+
+      - name: Install via kubectl as subchart in production mode and pass empty status informers
+        run: |
+          helm template test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false -f test-values.yaml | kubectl apply -f -
+          kubectl rollout status deployment test-chart --timeout=2m
+          kubectl rollout status deployment replicated --timeout=2m
+
+          COUNTER=1
+          while ! kubectl logs deploy/replicated | grep -qv 'Generated informers from Helm release'; do
+            ((COUNTER += 1))
+            if [ $COUNTER -gt 60 ]; then
+              echo "Did not receive empty status informers"
+              kubectl logs deploy/replicated
+              exit 1
+            fi
+            sleep 1
+          done
+
+      - name: Upgrade via kubectl as subchart in production mode to use default status informers
```

Comment:
is testing with helm template + kubectl apply necessary? feel like we're testing that the bug exists. ideally, status informers would also be automatically generated here, but it's currently a limitation.